radixdlt-core/radixdlt/src/main/java/org/radix/network/discovery/BootstrapDiscovery.java

```diff
@@ -17,55 +17,52 @@
 
```

Comment:
This seems unrelated to API. Would be better to have this in a different PR.

Reply:
Rolled back changes. For these minor changes I'll create a separate PR.